lib/link-active.ts

```diff
@@ -30,13 +30,16 @@ export interface LinkActiveOptions {
    @Input('linkActive') activeClass: string = 'active';
    @Input() activeOptions: LinkActiveOptions = { exact: true };
    private _sub: any;
+   private _linksSubscribed: {link: LinkTo, subscription: any}[] = [];
 
    constructor(
      @Query(LinkTo) public links: QueryList<LinkTo>,
      public element: ElementRef,
      public router$: Router,
      public renderer: Renderer
-   ) {}
+   ) {
+     this.links.changes.subscribe(_ => this.subscribeLinks());
```

Comment:
Move this out of the constructor into ngOnInit. We only want to use the constructor for wiring up dependencies